syntax = "proto2";
package gpb;
option optimize_for = LITE_RUNTIME;

/* Raft protocol messages, see include/rlite/raft.hpp */

message BwResRoute {
  required uint64 dst_addr = 1;
  required uint64 src_addr = 2;
  required uint32 dst_cepid = 3;
  required uint32 src_cepid = 4;
  required string next_hop = 5;
}
